README.md

```diff
@@ -84,7 +89,14 @@ install failed
 ERROR: Service 'php-cas' failed to build: The command '/bin/sh -c pear install -Z CAS-1.3.4.tgz  && rm /CAS-1.3.4.tgz' returned a non-zero code: 1
 ```
 You may just remove ` RUN pear install -Z CAS-1.3.4.tgz  && rm /CAS-1.3.4.tgz` from `php-cas/Dockerfile`.
-It will not affect SAML test bench
+It will not affect SAML test bench.
+
+If you run into ` mod_ssl error`, you may remove
+```
+RUN yum -y update \
+&& yum -y install php mod_ssl
+```
+from the file sp/DockerFile
```

Comment:
If it can cause an error and it works without it, why not modify the file for everyone?

SP project does not seem to use SSL anyway

Reply:
Done